internal/api/runme/parser/v1/parser.proto

```diff
@@ -51,6 +58,7 @@ message DeserializeResponse {
 
 message SerializeRequest {
   Notebook notebook = 1;
+  RunmeIdentity identity = 2;
```

Comment:
@pastuxso I'm pondering if we should have an "options" field here that "identity" will be part of. Make the distinction between message/data and config/options more obvious; what do you think?

Reply:
I like the idea. Are you thinking of something like a **`Params`** structure?"

Reply:
> @sourishkrout I like the idea, Are you thinking of something like a **`Params`** structure?"

As discussed out-of-band, let's go with `Options`.

Reply:
Done :white_check_mark: